packages/web-components/fast-components/src/avatar/avatar.styles.ts

```diff
@@ -0,0 +1,70 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-foundation";
+import {
+  accentFillRestBehavior,
+  heightNumber,	
+  neutralForegroundRestBehavior
+} from "../styles/index";
+
+export const AvatarStyles = css`
+  ${display("flex")} 
+  :host {
+    max-width: calc(${heightNumber} * 2.5px);
+    height: calc(${heightNumber} * 2.5px);
+    position: relative;
+  }
+
+  .link {
+    text-decoration: none;
+    color: var(--neutral-foreground-res);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    min-width: 100%;
+  }
+
+  .square {
+    border-radius: 5px;
```

Comment:
For square, assuming per the spec that this would map to the corner-radius design system value?

Reply:
Yes, that is correct.